redfish-core/lib/led.hpp

```diff
@@ -515,21 +535,29 @@ inline void
  *
  * @param[in] aResp     Shared pointer for generating response message.
  * @param[in] objPath   Object path
- * @param[in] locationIndicatorActive true or fasle
+ * @param[in] locationIndicatorActive true or false
+ * @param[in] ledState  Need to set indicator LED status
  *
  * @return None.
  */
-inline void
-    setLocationIndicatorActive(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
-                               const std::string& objPath,
-                               bool locationIndicatorActive)
+inline void setLocationIndicatorActive(
+    const std::shared_ptr<bmcweb::AsyncResp>& aResp, const std::string& objPath,
+    bool locationIndicatorActive, const std::string& ledState = "")
 {
     BMCWEB_LOG_DEBUG << "Set location indicator active";
 
+    if (!ledState.empty() && ledState != "Lit" && ledState != "Blinking" &&
```

Comment:
Can we write this more like 
https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/43761/15/redfish-core/lib/systems.hpp

That validatePowerMode function

Reply:
Has been updated to upstream

Reply:
@lkammath Can we pick this up?

Reply:
Yes. it is already picked up